src/app/Http/Controllers/CrudController.php

```diff
@@ -38,6 +38,10 @@ public function __construct()
                 $this->crud->request = $request;
                 $this->setup();
 
+                if(config('backpack.crud.setFromDB', false)) {
```

Comment:
Space after if.

```
if (config('backpack.crud.setFromDB', false) !== false)) {
    // FOO BAR
}
```

Might be more obvious.

Then again defaults with config always irk me when used like this :p but I suppose that's not too evil.